config/nginx.conf

```diff
@@ -42,6 +42,16 @@ http {
         init_worker_by_lua_block {
             local stats = require("stats")
             stats.init({host="127.0.0.1", port={{MFSERV_TELEGRAF_STATSD_PORT}}, delay=10})
+
+            {% for PLUGIN in PLUGINS %}
+                {% if PLUGIN.extra_lua_nginx_conf_string %}
+                    ##### BEGIN OF LUA EXTRA NGINX CONF FOR PLUGIN {{PLUGIN.name}} #####
+                    try {
+                        {{PLUGIN.extra_lua_nginx_conf_string}}
+                    } @fallback;
```

Comment:
I don't know this syntax `try { ... } @fallback`

But it sounds very interesting to me ! Do you have a link to share about this ?